diff --git a/toolchain/extensions/llvm.bzl b/toolchain/extensions/llvm.bzl
index 2814871..5a441ea 100644
--- a/toolchain/extensions/llvm.bzl
+++ b/toolchain/extensions/llvm.bzl
@@ -51,7 +51,7 @@ def _constraint_dict(tags, name):
 def _llvm_impl_(module_ctx):
     for mod in module_ctx.modules:
         if not mod.is_root:
-            fail("Only the root module can use the 'llvm' extension")
+            continue
         toolchain_names = []
         for toolchain_attr in mod.tags.toolchain:
             name = toolchain_attr.name
